lib/defaults/defaults.go

```diff
@@ -697,7 +697,7 @@ const (
 	// this seems like a sensible default. The setting can also be overridden at installation time.
 	ServiceSubnet = "100.100.0.0/16"
 	// PodSubnet is a subnet dedicated to the pods in the cluster
-	PodSubnet = "100.96.0.0/16"
+	PodSubnet = "100.96.0.0/14"
```

Comment:
Any downsides to making the default subnet even larger, say /12 or /11 to potentially allow to scale up to max K8s supported size and make it more future-proof?

Reply:
Not really, we'd have to move the start range which isn't a big deal because we service cidr starts at 100.100.0.0...

It's also not hard to change this default at any time, or pass to the installer, so whatever the team prefers.